src/wallet/wallet.cpp

```diff
@@ -3298,7 +3298,7 @@ bool CWallet::InitLoadWallet()
             CKey key;
             key.MakeNewKey(true);
             if (!walletInstance->SetHDMasterKey(key))
-                throw std::runtime_error("CWallet::GenerateNewKey(): Storing master key failed");
+                throw std::runtime_error("CWallet::InitLoadWallet(): Storing master key failed");
```

Comment:
There is no way for SetHDMasterKey to fail currently.

However if that changes in the future it would be best that the failure was handled cleanly.